settings/optional_config.json

```diff
@@ -4,11 +4,11 @@
     "show_search_term": "True",
     "customcmd_color": "27007A",
     "rich_embed": "on",
-    "default_status": "idle",
+    "default_status": "dnd",
```

Comment:
Please do not contaminate PRs with configuration files from your own environment.